Apps/phawswafv2/readme.html

```diff
@@ -0,0 +1,18 @@
+<!-- File: readme.html
+  Copyright (c) 2018-2021 Splunk Inc.
+
+  SPLUNK CONFIDENTIAL - Use or disclosure of this material in whole or in part
+  without a valid written license from Splunk Inc. is PROHIBITED.
+-->
+<html>
+<body>
+<h2>Asset Configuration</h2>
+There are two ways to configure an AWS WAF asset. The first is to configure the <b>access_key</b>, <b>secret_key</b> and <b>region</b> variables. If it is preferred to use a role and Phantom is running as an EC2 instance, the <b>use_role</b> checkbox can be checked instead. This will allow the role that is attached to the instance to be used. Please see the <a href="https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/iam-roles-for-amazon-ec2.html">AWS EC2 and IAM documentation</a> for more information.
+
+<p>Region parameter provided in the asset configuration parameter and region of the bucket which is created in AWS console must match, otherwise the user will get an InvalidLocationConstraint error.</p>
+<p>For the <b>Update bucket</b> action, API is unable to validate the KMS key. Hence, it is recommended to provide a valid KMS key in this action parameter otherwise it will affect the S3 bucket. e.g If we update the S3 bucket with the invalid KMS key and then run create object action on the bucket then the action will not work for encryption = NONE.</p>
```

Comment:
```suggestion
<p>For the <b>Update bucket</b> action, the API is unable to validate the KMS key. Hence, it is recommended to provide a valid KMS key in this action parameter, otherwise it will affect the S3 bucket. For example, if we update the S3 bucket with an invalid KMS key and run the 'create object' action on the bucket then the action will not work for encryption = NONE.</p>
```